File: DataAccess/Models/Players/Player.cs
namespace DataAccess.Models.Players
{
    public class Player
    {
        public int PlayerId { get; set; }
        public string UserName { get; set; }
        public Guid Salt { get; set; }
        public string PassHash { get; set; }
        public string PlayerName { get; set; }
        public int Level { get; set; }
        public int Exp { get; set; }
        public int StatPointsGained { get; set; }
        public int StatPointsUsed { get; set; }
    }
}

File: GameServer/TypeScript/Game/Shared/Api/ApiInterfaces/Player.ts
interface ILoginData {
	currentZone: number;
	playerData: IPlayerData;
}

interface IPlayerData {
	userName: string;
	playerName: string;
	level: number;
	exp: number;
	attributes: IBattlerAttribute[];
	selectedSkills: number[];
	statPointsGained: number;
	statPointsUsed: number;
}

interface IInventoryData {
	inventory: IInventoryItem[];
	equipped: IInventoryItem[];
}

interface ILogPreference {
	name: string;
	enabled: boolean;
}